Makefile

```diff
@@ -34,7 +34,7 @@ compile-validator-tool: setup  ## Compiles configuration validator tool
 
 .PHONY: build-docker-image
 build-docker-image: ## Build docker image for use in e2e tests
-	docker build . --file Dockerfile --tag xds-relay
```

Comment:
how about renaming this make rule to `build-e2e-tests-docker-image`?

Reply:
fixed